CMakeLists.txt

```diff
@@ -45,6 +45,14 @@ execute_process(COMMAND ${LLVM_CONFIG} "--cxxflags"
                 OUTPUT_VARIABLE LLVM_CXXFLAGS
                 OUTPUT_STRIP_TRAILING_WHITESPACE)
 
+if(WITH_TARGET MATCHES "s2e")
```

Comment:
It's strange why this is not inherited from libcpu.
See this line: https://github.com/S2E/libcpu/blob/master/CMakeLists.txt#L87